Tasks/AzureResourceGroupDeployment/Tests/L0.ts

```diff
@@ -887,7 +887,6 @@ describe('Azure Resource Group Deployment', function () {
         tr.run();
 
         try {
-            assert(tr.succeeded, "Task should have succeeded");
```

Comment:
The task will not pass because when we query to validateExtensionStatus(after addition of the extension), it would have substatus as failed.